src/Mod/TechDraw/App/DrawViewDimension.cpp

```diff
@@ -1237,55 +1237,37 @@ anglePoints DrawViewDimension::getAnglePointsTwoEdges(ReferenceVector references
         TechDraw::GenericPtr generic0 = std::static_pointer_cast<TechDraw::Generic>(geom0);
         TechDraw::GenericPtr generic1 = std::static_pointer_cast<TechDraw::Generic>(geom1);
         Base::Vector3d apex = generic0->apparentInter(generic1);
-        Base::Vector3d farPoint0;
-        Base::Vector3d farPoint1;
-        // pick the end of generic0 farthest from the apex
+
+        Base::Vector3d farPoint0{generic0->getEndPoint()};
         if ((generic0->getStartPoint() - apex).Length()
             > (generic0->getEndPoint() - apex).Length()) {
             farPoint0 = generic0->getStartPoint();
         }
-        else {
-            farPoint0 = generic0->getEndPoint();
-        }

```

Comment:
Removing the else clause, `farPoint0` can be null. Could this cause misbehaviour?

Reply:
farPoint0 is set just above the if.